ocaml/xapi/quicktest_raw_vdi_import.ml

```diff
@@ -0,0 +1,57 @@
+(*
+ * Copyright (C) 2017 Citrix Systems Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+
+open Sparse_encoding
+open Client
+open Quicktest_common
+open Http
+
+let http request f =
+  let open Xmlrpc_client in
+  let transport =
+    if !using_unix_domain_socket
+    then Unix Xapi_globs.unix_domain_socket
+    else SSL(SSL.make ~use_fork_exec_helper:false (), !host, 443) in
+  with_transport transport (with_http request f)
+
+let import_raw_vdi ~session_id ~task_id f =
+  let req = Http.Request.make ~version:"1.0"
+      ~user_agent:"quicktest"
+      ~query:[
+        "session_id", Ref.string_of session_id;
+        "sr_id","fake_sr";
+        "task_id", Ref.string_of task_id;
+        "chunked", "true"
```

Comment:
Do we need this parameter in the query? - I think passing this parameter is not important for this test.